### ISSUE TITLE
LICENSE

### ISSUE THREAD

**Issue by J3RN** [CWDG/cwdg.github.io#20]
Well, I messed up and forgot to license this site. The people who have contributed since this version's inception appear to be just three of us. However, in the long and illustrious history of the site, there are several other names.

I am planning on licensing this site under the [MIT License](https://opensource.org/licenses/MIT).

At bare minimum, I need the consent of these people:
- [x] @benlk
- [x] @vipermark7
- [x] @J3RN

With those, I can rebase away all the history and have a clean codebase. To keep the long and illustrious history, I need the consent of:
- [x] @tarebyte
- [x] @saterus 
- [x] @RyanMcG 
- [x] @oslerw 
- [ ] @tonywok 
- [x] @jpeddicord
- [x] @scssquatch 

If you got tagged in this and your name is not checked off on the list, please comment expressing you accept the new MIT license.

**Comment by moore3071** [comment]
@J3RN, note that those tagged under the long and illustrious history had their content previously under MIT (with the old version of the site) and therefore their contributions are already licensed under MIT.

**Comment by J3RN** [comment]
@moore3071 Not true. The osurb site had only three contributors. Many listed here (tarebyte for instance) did not contribute to the osurb.github.io site, only to the cwdg.github.io fork of it.

**Comment by vipermark7** [comment]
I accept the new license :)

**Comment by jpeddicord** [comment]
I'm OK with MIT (or any license you choose), as I imagine most others are. :)

**Comment by tarebyte** [comment]
:shipit:

**Comment by vipermark7** [comment]
what is that? looks kind of like general ackbar with a bowler hat

**Comment by tarebyte** [comment]
@vipermark7 it's the ship it squirrel.

**Comment by vipermark7** [comment]
oh ok!

**Comment by benlk** [comment]
I am in favor of the MIT license.

**Comment by J3RN** [comment]
Alright, at this point I can rebase and blow away the history and have a clean repo, but I'd prefer not to if I don't have to. I'll give it a little while.

**Comment by scssquatch** [comment]
You have my consent :)

On Mon, May 09, 2016 at 11:46 AM vipermark7

<
mailto:vipermark7 <notifications@github.com>
> wrote:

oh ok!

**Comment by saterus** [comment]
Absolutely. I have no idea what I did for this site, so it's all fair game. I'm actually surprised we removed the MIT license. That was almost certainly a mistake. Feel free to blame me on that one.

**Comment by J3RN** [comment]
@saterus It's all a little weird. This repo (cwdg.github.io) never seems to have been under a license. However, the cwdg-website repo was, mercifully, always licensed under MIT. This repo now holds a static port of the Rails app that was in the cwdg-website repo, for the sole reason of the name of the repo and using GitHub pages for hosting.

Since this new version contains none of the code from what used to be here, this whole "having old contributors agree to a license" thing seems a bit heavy-handed, but I suppose adding the license means licensing the code in the git history, in addition to the present and future code.

## TL;DR
### Thanks! :smiley:

**Comment by oslerw** [comment]
My contributions are MIT, thanks

**Comment by RyanMcG** [comment]
@J3RN LGTM

@tarebyte we use :rocket: to ship at Lyft. Nice rodent though. :wink:

**Comment by J3RN** [comment]
Alright, Tony's not heeding the call, but that's alright. The old repo (osurb.github.io) from which his commits originate is now under MIT, and that should qualify that work, also contained here, as being under MIT. If he decides he wants to sue me... Well, I hope he doesn't.

Anyhow, thank all of you for being awesome. Adding MIT license immediately. :smile:

**Comment by tonywok** [comment]
Sorry guys for being late - wasn't checking my notifications. MIT all the way. Consent!
